libsoftwarecontainer/unit-test/softwarecontainerlib_unittest.cpp

```diff
@@ -88,6 +89,21 @@ TEST_F(SoftwareContainerApp, TestWayland) {
 
 }
 
+TEST_F(SoftwareContainerApp, DoubleIDCreatesError) {
```

Comment:
We are bad at commenting/documenting our tests, but I think a few lines documentation would be nice here, why are we doign this test and what's the expected behavior.